docker/Dockerfile

```diff
@@ -4,67 +4,55 @@
 ## Copyright (c) 2015-2020 ODC Contributors
 ## SPDX-License-Identifier: Apache-2.0
 ##
-ARG ORG=opendatacube
-ARG V_BASE=3.3.0
 ARG V_PG=12
-FROM ${ORG}/geobase-builder:${V_BASE} as env_builder
+FROM osgeo/gdal:ubuntu-small-latest as env_builder
+
+# Update and install Ubuntu packages
+
+USER root
+RUN apt-get update -y \
+    && DEBIAN_FRONTEND=noninteractive apt-get upgrade -y \
+    && DEBIAN_FRONTEND=noninteractive apt-get dist-upgrade -y \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-change-held-packages --fix-missing --no-install-recommends \
+        git \
+        libpq-dev libudunits2-dev \
+        python3-dev python3-distutils python3-pip \
+        build-essential \
+        postgresql \
+        postgresql-client-${V_PG} \
+        postgresql-${V_PG} \
+        sudo make graphviz \
+        tini \
```

Comment:
I found the Ubuntu version of `tini` didn't work as expected. Do we need tini anyway? It's usually for a running service, whereas this is just a test environment.

Reply:
As noted below, I'm not 100% sure if we need tini.  We are running a Postgres server and the datacube tests in the one Docker image though, so maybe it allows the postgres server to run more safely in the background?

Reply:
My view is that we should be decoupling Postgres and the ODC test image. Using Docker Compose makes running a postgres DB in its own container easy and I think it's better.

That could be a separate PR though.